assets/global.js

```diff
@@ -547,7 +547,7 @@ class SliderComponent extends HTMLElement {
 
   update() {
     const previousPage = this.currentPage;
-    this.currentPage = Math.round(this.slider.scrollLeft / this.sliderLastItem.clientWidth) + 1;
+    this.currentPage = Math.round(this.slider.scrollLeft / this.sliderItemOffset) + 1;
```

Comment:
Something I noticed here because I might change the amount of slides with scroll by. Right now it's 1. Though we have `step` that is created and used in the`onButtonClick(event)` function. But when adding `data-step="2"` to test, the page count isn't changing properly.
So I'm thinking we could make `step` a variable in the constructor and maybe the attribute can be on the slider wrapper rather than each button. 
And then we use `step` in this calculation as well. So that it looks something like: 
```javascript
this.currentPage = Math.round(this.slider.scrollLeft / (this.sliderItemOffset * step)) + 1;
```

I realize it isn't quite the scope of this PR but thought I'd mention it. It doesn't seem like too big of a thing to fix. Which I could do in my `feat. collection` PR since that's where I most likely will need to use `data-step`. 

and I guess the amount of total page should be change accordingly 🤔